exotica_python/src/pyexotica/interactive_rho_tuning.py

```diff
@@ -0,0 +1,33 @@
+import Tkinter
```

Comment:
New run dependency not included in the package.xml

Reply:
I've added Tkinter to `package.xml` as advised however it is in standard library for Python (see [here](https://docs.python.org/2/library/tk.html)) so was this really necessary?